yt_dlp/extractor/weibo.py

```diff
@@ -192,8 +192,30 @@ class WeiboIE(WeiboBaseIE):
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        return self._parse_video_info(self._weibo_download_json(
-            f'https://weibo.com/ajax/statuses/show?id={video_id}', video_id))
+        meta = self._weibo_download_json(f'https://weibo.com/ajax/statuses/show?id={video_id}', video_id)
+        mix_media_info = traverse_obj(meta, ('mix_media_info', 'items'))
+        if not mix_media_info:
+            return self._parse_video_info(meta)
+
+        playlist_title = traverse_obj(mix_media_info, (0, 'data', 'content1'))
+        return self.playlist_result(self._entries(mix_media_info), playlist_title=playlist_title)
```

Comment:
I'd rather return just an id and no title instead of arbitrarily using the first entry's title
```suggestion
        return self.playlist_result(self._entries(mix_media_info), video_id)
```